src/portal/src/i18n/lang/en-us-lang.json

```diff
@@ -959,8 +957,11 @@
             "OIDC_SETNAMECONTENT": "You must create a Harbor username the first time when authenticating via a third party(OIDC).This will be used within Harbor to be associated with projects, roles, etc.",
             "OIDC_USERNAME": "Username",
             "GROUP_CLAIM_NAME": "Group Claim Name",
+            "GROUP_CLAIM_NAME_INFO": "The name of a custom group claim that you have configured in your OIDC provider",
             "OIDC_ADMIN_GROUP": "OIDC Admin Group",
-            "OIDC_ADMIN_GROUP_INFO": "Specify an OIDC admin group name. All OIDC users in this group will have harbor admin privilege. Keep it blank if you do not want to."
+            "OIDC_ADMIN_GROUP_INFO": "Specify an OIDC admin group name. All OIDC users in this group will have harbor admin privilege. Keep it blank if you do not want to.",
+            "OIDC_GROUP_FILTER": "OIDC Group Filter",
+            "OIDC_GROUP_FILTER_INFO": "The OIDC group filter which filters out the groups that don't match the regular expression."
```

Comment:
It is not clear what type or standards of regular expression are allowed?
Is it this one?

Thre should be reference.
https://pkg.go.dev/regexp/syntax

Reply:
Yes,  it's that one.
But, It is not suitable to put the reference in the tooltip. Will explain it in the doc.